core/rpc/src/rpc_impl/query.rs

```diff
@@ -26,25 +32,132 @@ where
 {
     pub(crate) fn inner_get_balance(
         &self,
-        udt_hashes: Vec<Option<H256>>,
+        udt_hashes: HashSet<Option<H256>>,
+        address: QueryAddress,
+        block_number: Option<u64>,
```

Comment:
// Not support yet, so we should return error, when the block_number is `none`, which is missed now